source/extensions/filters/http/cache/cacheability_utils.cc

```diff
@@ -69,8 +69,17 @@ bool CacheabilityUtils::isCacheableResponse(const Http::ResponseHeaderMap& heade
       (headers.Date() && response_cache_control.max_age_.has_value()) ||
       headers.get(Http::Headers::get().Expires);
 
+  // TODO(cbdm): only allowing responses to be cacheable if there's no vary header or it is empty;
+  // need to implement the logic for using an allowlist.
+  // TODO(cbdm): "vary: accept-encoding" could be treated in the cache; we can encode (decode) the
+  // content to the accepted format the user specified. This way we wouldn't need to store different
+  // versions for this header, and could simply convert to that format and serve.
```

Comment:
s/this header/this payload/ ?

Reply:
Just making sure I understand the suggestion: replace `versions for this header` with `versions for this payload`?
I agree it's better, just wanted to be sure before changing it.

Reply:
Gentle ping here.

Reply:
confirmed; that was the suggested rewording.